### DISCUSSION TITLE
Question(mux-player-react): Is it possible to hide the player controls.

### DISCUSSION THREAD

**Discussion by tronic-chris** [muxinc/elements#315] [Q&A] [answered]
Is it possible to hide the video player controls similar to the `controls` prop in `mux-video-react`?

**Comment by dylanjha** [comment]
@tronic-chris not currently, but we're looking at ways to customize the existing controls. Stay tuned for more updates.

**Comment by cabello** [comment]
In our use case we use VOD as part of a live experience. So we tell a rich client to load the same VOD at the same time, the `streamType='live'` is the closest we have in terms of UI, but I would like to hide the pause button, only show the play button if the browser decided to block the autoplay of that video.
<img width="117" alt="CleanShot 2022-08-05 at 09 49 00@2x" src="https://user-images.githubusercontent.com/60936/183090882-c61c6b30-2415-428a-9fc0-a1e24951e6be.png">

Also probably hide the live icon:
<img width="87" alt="CleanShot 2022-08-05 at 09 48 22@2x" src="https://user-images.githubusercontent.com/60936/183090803-5ab9386d-36b1-47e6-9310-79c9fea3daaa.png">

**Comment by luwes** [comment, marked as answer]
This is possible since Mux Player beta.27 🎉 Here is the info around hiding controls with CSS vars:
- https://github.com/muxinc/elements/releases/tag/%40mux%2Fmux-player%400.1.0-beta.27
- https://github.com/muxinc/elements/tree/main/packages/mux-player#hiding-controls
